Modules/ThirdParty/GDCM/UpdateFromUpstream.sh

```diff
@@ -11,6 +11,7 @@ readonly repo="https://github.com/malaterre/GDCM.git"
 readonly tag="release"
 readonly shortlog=false
 readonly paths="
+  .gitattributes
```

Comment:
Maybe `.gitattributes` gets overwritten on lines 44-45 in this file (UpdateFromUpstream.sh)? The [file in GDCM](https://github.com/malaterre/GDCM/blob/v3.0.10/.gitattributes) is pretty short, the [one in ITK](https://github.com/InsightSoftwareConsortium/ITK/blob/v5.3rc03/Modules/ThirdParty/GDCM/src/gdcm/.gitattributes) is longer, so the merge does not result in any changes to this file?

Reply:
Yes. You can use `git commit -n` to avoid the local check though (which is why the robot's enforcement is the trusted one). You may need to modify the update-common script to add the `-n` flag (no need to commit it).

And yes, removing the check for it would be handy. Still need to do it in VTK too…